packages/angular/projects/ui-angular/src/lib/primitives/phone-number-field/phone-number-field.component.html

```diff
@@ -19,9 +19,9 @@
     >
       <amplify-form-select
         name="country_code"
-        label="Country Code"
+        label="Country code"
```

Comment:
This is another one of those weird inconsistencies where in React, and Vue it's `Country code` but in Angular it's `Country Code`. I'll put it back to `Country Code` and just use a regular expression in the cypress test to match it, in case someone is using i18n

Reply:
Updated back to `Country Code` in beffa1f073ca69d2a831ba7a98f6eba4b59919db